examples/morpheusvm/tests/e2e/transfer_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package e2e_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/vm"
+	"github.com/ava-labs/hypersdk/tests/workload"
+
+	he2e "github.com/ava-labs/hypersdk/tests/e2e"
+	ginkgo "github.com/onsi/ginkgo/v2"
+)
+
+var _ = he2e.RegisterTest("Transfer Transaction", func(t ginkgo.FullGinkgoTInterface, tn workload.TestNetwork) error {
+	require := require.New(t)
+
+	uri := tn.URIs()[0]
+	other, err := ed25519.GeneratePrivateKey()
+	require.NoError(err)
+	aother := auth.NewED25519Address(other.PublicKey())
+
+	lcli := vm.NewJSONRPCClient(uri)
+	parser, err := lcli.Parser(context.Background())
+	require.NoError(err)
+
+	spendingKey, err := tn.WorkloadFactory().GetSpendingKey()
+	require.NoError(err)
+
+	cli := jsonrpc.NewJSONRPCClient(uri)
+	_, tx, _, err := cli.GenerateTransaction(
+		context.Background(),
+		parser,
+		[]chain.Action{&actions.Transfer{
+			To:    aother,
+			Value: 1,
+		}},
+		auth.NewED25519Factory(ed25519.PrivateKey(spendingKey.Bytes)),
+	)
+
+	require.NoError(err)
+
+	timeoutCtx, timeoutCtxFnc := context.WithDeadline(context.Background(), time.Now().Add(2*time.Second))
```

Comment:
we have a const `txCheckInterval`. I think it makes sense to either pass the const in here or remove the const completely and pass in a value from the test.